node_modules/gh-admins/lib/api.js

```diff
@@ -160,6 +160,39 @@ var updateGlobalAdmin = module.exports.updateGlobalAdmin = function(ctx, id, dis
 };
 
 /**
+ * Change a global administrator's password
+ *
+ * @param  {Context}        ctx                     Standard context containing the current user and the current app
+ * @param  {string}         id                      The id of the global administrator for which to change the password
+ * @param  {string}         newPassword             The new password for the global administrator
+ * @param  {Function}       callback                Standard callback function
+ * @param  {Object}         callback.err            An error that occurred, if any
+ */
+var changePassword = module.exports.changePassword = function(ctx, id, newPassword, callback) {
+    // Ensure the global administrator id is a valid number
+    id = GrasshopperUtil.getNumberParam(id);
+
+    // Validation
+    var validator = new Validator();
+    validator.check(null, {'code': 401, 'msg': 'Only global administrators can change the password of a global administrator'}).isGlobalAdmin(ctx);
+    validator.check(id, {'code': 400, 'msg': 'A valid global administrator id must be provided'}).isInt();
+    validator.check(newPassword, {'code': 400, 'msg': 'A valid password must be provided'}).notEmpty();
+    validator.check(newPassword, {'code': 400, 'msg': 'A valid password that is 6 characters or longer must be provided'}).len(6);
+    if (validator.hasErrors()) {
+        return callback(validator.getFirstError());
+    }
+
+    // Verify that the provided global administrator exists
+    getGlobalAdmin(ctx, id, function(err, globalAdmin) {
+        if (err) {
+            return callback(err);
+        }
+
+        return AdminsDAO.changePassword(globalAdmin, newPassword, callback);
+    });
```

Comment:
I feel like this kind of endpoint should also take the current password of the global admin in context, just to ensure this operation is not available for a global admin whose session was hijacked somehow (e.g., some CORS issue, XSS, etc...)